contracts/exchange-libs/contracts/src/MixinLibMathRichErrors.sol

```diff
@@ -0,0 +1,36 @@
+pragma solidity ^0.5.9;
+
+import "@0x/contracts-utils/contracts/src/RichErrors.sol";
+import "./interfaces/IMixinLibMathRichErrors.sol";
+
+
+contract MixinLibMathRichErrors is
+    IMixinLibMathRichErrors,
+    RichErrors
+{
+    // solhint-disable func-name-mixedcase
+    function DivisionByZeroError()
+        internal
+        pure
+        returns (bytes memory)
+    {
+        return abi.encodeWithSelector(DIVISION_BY_ZERO);
```

Comment:
Since this takes no arguments, it should be cheaper to hard code the return value.